assets/app/view/game/player.rb

```diff
@@ -38,6 +39,7 @@ def render
         if @show_companies
           divs << h(Companies, owner: @player, game: @game, show_hidden: @show_hidden) if @player.companies.any? || @show_hidden
           divs << h(UnsoldCompanies, owner: @player, game: @game) unless @player.unsold_companies.empty?
+          divs << h(HiddenHand, player: @player, game: @game, user: @user) if @hidden_hand
```

Comment:
This currently renders `HiddenHand` on player cards for every game. You need some way to gate this for only 18 India. Maybe instead of `hidden_hand` being a need for this view, this line checks a new method on the game, and it works basically like the method `show_ipo_rows?` you're adding here.

Reply:
I think my original idea was to check if `hand_companies_for_stock_round` was empty, but a boolean flag in the game/base file seems more straightforward, as you suggested.